src/main/java/org/kohsuke/github/GHPullRequestReviewBuilder.java

```diff
@@ -75,19 +75,53 @@ public GHPullRequestReviewBuilder event(GHPullRequestReviewEvent event) {
      * Comment gh pull request review builder.
      *
      * @param body
-     *            The relative path to the file that necessitates a review comment.
+     *            Text of the review comment.
      * @param path
+     *            The relative path to the file that necessitates a review comment.
+     * @param position
      *            The position in the diff where you want to add a review comment. Note this value is not the same as
      *            the line number in the file. For help finding the position value, read the note below.
-     * @param position
-     *            Text of the review comment.
      * @return the gh pull request review builder
      */
     public GHPullRequestReviewBuilder comment(String body, String path, int position) {
```

Comment:
Deprecated?

Reply:
In the documentation of the GitHub API (https://docs.github.com/en/rest/pulls/reviews?apiVersion=2022-11-28#create-a-review-for-a-pull-request), the use of position when adding a comment at the moment we create a review is not marked as deprecated (surprisingly), that's why I kept this not depreacted.

The position parameter is only marked deprecated when we create a comment on a pull request review (https://docs.github.com/en/rest/pulls/comments?apiVersion=2022-11-28#create-a-review-comment-for-a-pull-request). In doubt, I just followed the current documentation.

Reply:
@bitwiseman Do you still want to mark this function as deprecated or leave it as is for now? (despite the confusing GH documentation on this point...)

Reply:
Nah, leave it.